ir/base.def

```diff
@@ -266,6 +285,14 @@ class Annotation {
 
     /// Useful helper: extracts name value from a name annotation
     static cstring getName(const Annotation* annotation);
+
+    AnnotationKind annotationKind() const {
```

Comment:
@antoninbas : here is a helper function which may be useful.

Reply:
based on the latest proposals for both specifications, it seems that we really have 4 cases: unstructured, structured empty, structured kv list and structured list. However, I think the implementation can be adjusted later and we can merge this as it is.